src/applications/common/schemaform/fields/ArrayField.jsx

```diff
@@ -251,11 +270,11 @@ export default class ArrayField extends React.Component {
                           disabled={disabled}
                           readonly={readonly}/>
                       </div>
-                      {notLastOrMultipleRows &&
+                      {!isAddingFirst &&
```

Comment:
@markgreenburg actually we do already provide the remove buttons to added items (but we did not previously provide the update button to the last item -- which needed to be updated as we now allow last items to be in edit mode), I've fixed the bug here. Nice catch!